src/filesystem/implementation.js

```diff
@@ -48,28 +48,28 @@ const { validateInteger } = require('../shared.js');
 function update_node_times(context, path, node, times, callback) {
   // Honour mount flags for how we update times
   var flags = context.flags;
-  if(_(flags).contains(FS_NOCTIME)) {
+  if (_(flags).contains(FS_NOCTIME)) {
```

Comment:
It looks like your editor has done a ton of formatting changes in this file.  We should undo that, so we don't lose all our git history.

My suggestion to you is to do the following:

1. copy/paste your changes from this file so you don't lose them.
1. `git checkout master src/filesystem/implementation.js`
1. re-add your copy/pasted changes from `src/filesystem/implementation.js` back to the file
1. add, commit, push this back up.